src/actions/auth.js

```diff
@@ -23,15 +23,19 @@ exports.registrationFailure = (errMessage) => ({
     payload: errMessage
 });
 
+exports.loginFailure = (errMessage) => ({
+    type: AuthAct.LOGIN_FAIL,
+    payload: errMessage
+});
+
+
 exports.registerUser = ({ email, password, firstName, lastName }) => {
 
     return (dispatch) => {
 
         dispatch(actions.registrationRequest({ email, password, firstName, lastName }));
 
-        const newUser = WebClient.post('/users', { email, password, firstName, lastName });
-
-        newUser
+        return WebClient.post('/users', { email, password, firstName, lastName })
```

Comment:
Actions have been constructed in this manner because it's supposed to make testing these actions easier.  Not that we're currently doing any of that, but it is good to set ourself up for it when we make the jump.